setup.py

```diff
@@ -559,6 +559,9 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 zendesk = [
     'zenpy>=2.0.24',
 ]
+zeppelin = [
```

Comment:
This can be removed as pyzeppelin is in this PR

Reply:
Will fix it. Thx for the suggestion.